Automattic/wp-calypso
# Pre-Release Tests: Skip Woo google auth flow steps until recaptcha is accounted for

Related to p1719199948669409/1719199255.061099-slack-C02DQP0FP

## Proposed Changes

There is a new recaptcha screen from google that needs to be handled before a password can be entered, so we comment out all test logic after and including the entering the user's password p1719199948669409/1719199255.061099-slack-C02DQP0FP

## Why are these changes being made?


* Pre release tests are failing because there were updates made by a 3rd party to an external service, so our e2e specs no longer reflect a successful auth flow

## Testing Instructions



* Set up e2e specs in a local dev environment ( see https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/README.md )
* Run `yarn workspace wp-e2e-tests test -- test/e2e/specs/onboarding/signup__woo-google.ts`

## Pre-merge Checklist
